tensorflow/python/ops/init_ops.py

```diff
@@ -138,8 +141,8 @@ def constant_initializer(value=0, dtype=dtypes.float32):
     >>> with tf.Session():
     >>>   x = tf.get_variable('x', shape=[2, 3], initializer=init)
 
-    ValueError: Too many elements provided. Needed at most 6, but received 8
-    ```
+    ValueError: Too many elements provided. Needed at most 6, but received 8.
```

Comment:
The extra space caused the bug. It's easy to test in the markdown editor, and it would be considered as `code` instead of quote flag if you give the extra space.